### DISCUSSION TITLE
ASUS AI Accelerator PCIe Card

### DISCUSSION THREAD

**Discussion by ehn** [blakeblackshear/frigate#2325] [Ask A Question]
Does Frigate work with the ASUS [AI Accelerator PCIe Card](https://iot.asus.com/products/AI-accelerator/AI-Accelerator-PCIe-Card/)? Is anyone using this setup?

**Comment by blakeblackshear** [comment]
It works, but it's massive overkill. An M1 Mac Mini is much cheaper and also has equivalent performance to a Coral on each neural core.
